static/bitcoin_sentiments.js

```diff
@@ -18,21 +18,42 @@ angular.module('BitcoinSentimentsApp', [])
       {'id_str': 19111989189, 'created_at': '2025-03-03 01:00:01', 'full_text': 'Tweet number 2', 'subjectivity': 0.5, 'polarity': 0.8}
     ];
 
-    tweetTable.getTweets = function() {
-      tweetTable.tweets = [
-        {'id_str': 1, 'full_text': 'working!'}
-      ]
-
-      BitcoinSentimentsService.getTweetsFromServer()
-      .then(function(response) {
-        return response.json()
-      }).then(function(data) {
-        console.log('getTweetsFromServer')
-        Bokeh.embed.embed_item(data['plot2'], 'mainplot_div')
-        tweetTable.tweets = data['tweets']
-        $scope.$apply();
-        console.log('getTweetsFromServer finished.')
-      })
+    tweetTable.showTimeline = function() {
+        if($('#card_li_plot a').hasClass('bs_timeline_empty') && $('#card_li_intro a').hasClass('active')) {
+            $('#card_li_intro a').removeClass('active')
+            $('#card_li_plot a').addClass('active')
+            $('#card_body_intro').css('display', 'none')
+            $('#card_body_plot').css('display', 'block')
+
+            tweetTable.tweets = [
+                {'id_str': 1, 'full_text': 'working!'}
+            ]
+
+            BitcoinSentimentsService.getTweetsFromServer()
+            .then(function(response) {
+                return response.json()
+            }).then(function(data) {
+                $('#plot_card_spinner').css('display', 'none')
+                Bokeh.embed.embed_item(data['plot2'], 'mainplot_div')
```

Comment:
Include timezone for the time axis label.

Reply:
Fixed in [01c0714](https://github.com/fund3/sentiment-dashboard/pull/4/commits/01c0714ebbc5e4495f9523b09bb94b9ea89d1b7a).